.github/workflows/delete-model-runs.yaml

```diff
@@ -0,0 +1,58 @@
+# Workflow that can be manually dispatched to delete test model runs that
+# do not need to be persisted indefinitely.
+#
+# Gated such that it's impossible to delete runs older than the current
+# assessment cycle, where each assessment cycle starts in April.
+
+name: delete-model-runs
+
+on:
+  workflow_dispatch:
+    inputs:
+      run-ids:
+        description: >
```

Comment:
**suggestion:** IMO it'd be really helpful here to have a default input showing an example (nonexistent) run ID.

Reply:
True, done in a4895c20d2b44cad197df855ac7cdbeebbbe9616! Note that the trade-off here is that default arguments are populated as actual values in the workflow input form, rather than placeholder text, so there's a chance of a caller misinterpreting the form and submitting it with the (fake) default values. This should just raise an error and not actually delete any data, however, so the risk is worth it in my view.